cocos/core/scene-graph/node.ts

```diff
@@ -562,11 +562,23 @@ export class Node extends BaseNode {
      */
     public setPosition (x: number, y: number, z: number): void;
 
-    public setPosition (val: Vec3 | number, y?: number, z?: number) {
-        if (y === undefined || z === undefined) {
+    
+    /**
+     * @en Set position in local coordinate system
+     * @zh 设置本地坐标
+     * @param x X axis position
+     * @param y Y axis position
+     */
+    public setPosition (x: number, y: number): void;
```

Comment:
If position support 2d API, then scale and eulerAngles should too

Reply:
fixed in commit https://github.com/cocos-creator/engine/pull/7610/commits/ed35eb2e8e16cbe430e8f77bce7ec3ff5cf7b70d